internal/handlers/common/params.go

```diff
@@ -93,3 +93,89 @@ func GetNumberParam(parameterName string, value any) (int64, error) {
 	}
 	return numberValue, nil
 }
+
+// GetBinaryMaskParam returns *types.Binary value matched and error if match failed.
+// Possible values are: position array ([1,3,5] == 010101), integer value and types.Binary value.
+func GetBinaryMaskParam(value any) (mask *types.Binary, err error) {
```

Comment:
Let's return types.Binary by value as we do with all scalar types (even if that's a small structure).

Let's also do not use named results if we don't actually use them